src/components-examples/cdk/tree/cdk-tree-flat/cdk-tree-flat-example.ts

```diff
@@ -88,7 +88,13 @@ export class CdkTreeFlatExample {
   }
 
   shouldRender(node: ExampleFlatNode) {
-    const parent = this.getParentNode(node);
-    return !parent || parent.isExpanded;
+    let parent = this.getParentNode(node);
+    while (parent) {
+      if (!parent.isExpanded) {
+          return false;
```

Comment:
This indentation still seems a bit too much. We use 2 spaces everywhere.

Reply:
My bad again. I changed the configuration files on my IDE. Hope it is fine now! [5dfafba](https://github.com/angular/components/commit/5dfafba19c4612c8b91a9421fee945b8b02ce1b6)